p2p/src/net/libp2p/proto/sync.rs

```diff
@@ -137,8 +140,6 @@ impl Backend {
 
 #[cfg(test)]
 mod tests {
```

Comment:
Maybe either remove this or add a TODO and explain what tests have to be added later